### ISSUE TITLE
how to loop through inventory for custom dns

### ISSUE THREAD

**Issue by Dialgatrainer02** [ngine-io/ansible-role-blocky-dns#2]
whn using this role how do i set up custom dns to have dns names to all the machines in my inventory using there ansible_host as there ip address and invntory_hostname + domain name using the custom dns var. This was my attempt but its not valid yaml

```yml
- name: Setup dns servers
  hosts: dns
  roles:
    - role: ngine_io.blocky_dns
  vars:
    blocky__custom_dns: {% for host in groups["all"] %} ${host} .domain.com: {{ hostvars[host].ansible_host | ansible.utils.ipv4 }} {%% endif %}

```
so it would expand to 
```yml
- name: Setup dns servers
  hosts: dns
  roles:
    - role: ngine_io.blocky_dns
  vars:
    blocky__custom_dns:
      grafana.domain.com: 192.168.0.100
      nginx.domain.com: 192.168.0.101
      etc

```

**Comment by resmo** [comment]
Hi @Dialgatrainer02 thanks for this question, 

I thought about it and I am going to implement an option to make an integrated way (easier way) to add all hosts into the custom dns beside the possibility to also add more custom dns using `blocky__custom_dns`

**Comment by Dialgatrainer02** [comment]
Cool thank you that would be perfect

**Comment by resmo** [comment]
implemented in #3
